packages/driver/cypress/integration/commands/files_spec.js

```diff
@@ -394,7 +443,7 @@ describe('src/cy/commands/files', () => {
       Cypress.backend.resolves(okResponse)
 
       cy.writeFile('foo.txt', 'contents').then((subject) => {
-        expect(subject).to.not.exist
+        expect(subject).to.eq(null)
```

Comment:
Made this more explicit. I had a bug where I was yielding undefined and this test continued to pass.